### PR TITLE
chore: update My Programs aesthetics

### PR DESCRIPTION
**Pull request by t-proctor**
##### Description

Updated the icon color and bottom background color of programs card
##### Refers/Fixes

#292 
##### Testing
Before
Visual testing

### CODE REVIEW

File: packages/round-manager/src/features/program/ListProgramPage.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ function ListPrograms() {
         floatingIcon={
           <div
             className="relative flex justify-center items-center h-16 w-16 translate-x-6 -translate-y-8 bg-teal-500 text-white rounded drop-shadow-xl">
-            <RefreshIcon className="h-6 w-6" aria-hidden="true" />
+            <RefreshIcon className="h-6 w-6 text-gray-500" aria-hidden="true" />
```
Comment (chibie) [inline packages/round-manager/src/features/program/ListProgramPage.tsx:89]: color code should be `text-grey-500` not `text-gray-500`

closely related... ikr

**Review by chibie — CHANGES_REQUESTED** [review]

File: packages/round-manager/src/features/program/ListProgramPage.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const ProgramCard: React.FC<ProgramCardProps> = (props: ProgramCardProps) => (
       </CardDescription>
     </CardContent>
     <CardFooter>
-      <CardFooterContent className="justify-end p-6">
+      <CardFooterContent className="justify-end p-6 bg-[#F3F3F5]">
```
Comment (chibie) [inline packages/round-manager/src/features/program/ListProgramPage.tsx:43]: it's the white area behind the card that should be bg-grey-150 (#F3F3F5) not the card footer.

could you also make this change?

we could also get on a call to explain this

**Review by chibie — DISMISSED** [review]

File: packages/round-manager/src/features/program/ListProgramPage.tsx (hunk 2 above)
Comment (t-proctor) [reply packages/round-manager/src/features/program/ListProgramPage.tsx:43]: Did you mean this area?
fixed

Comment (chibie) [reply packages/round-manager/src/features/program/ListProgramPage.tsx:43]: exactly! that area

Comment (t-proctor) [reply packages/round-manager/src/features/program/ListProgramPage.tsx:43]: I'm free to hop on call to clear up my confusion also

**Review by shavinac — APPROVED** [review]

**Comment by shavinac** [comment]
FYI if you put `fixes #292` in the PR description, GitHub will automatically link the PR to the issue
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Review by thelostone-mc — APPROVED** [review]

**Review by nfrgosselin — APPROVED** [review]
LGTM — thanks @t-proctor!

**Pull request merged**